torch/_inductor/fx_passes/post_grad.py

```diff
@@ -1207,6 +1207,15 @@ def view_to_reshape(gm):
     ):
         nd.target = torch.ops.aten.reshape.default
 
+    subgraph_names: OrderedSet[str] = OrderedSet()
```

Comment:
more of a general question - we're probably going to have to audit the other one-off passes that inductor runs and ensure that they recurse too, right? I wonder if we can add some more invariants / refactor inductor's one-off passes so that they get this recursive behavior more automatically, for future graph-pass-writers.

Reply:
@eellison This one is run outside of post_grad, so we dont use GraphTransformObserver here.